src/platform/user/profile/vet360/containers/VAPProfileField.jsx

```diff
@@ -298,14 +320,20 @@ export const mapStateToProps = (state, ownProps) => {
   const data = selectVet360Field(state, fieldName);
   const isEmpty = !data;
   const addressValidationType = selectAddressValidationType(state);
+  const activeEditView = selectCurrentlyOpenEditModal(state);
   const showValidationView =
     ownProps.ValidationView &&
     addressValidationType === fieldName &&
     // TODO: use a constant for 'addressValidation'
-    selectCurrentlyOpenEditModal(state) === 'addressValidation';
+    activeEditView === 'addressValidation';
 
   return {
     analyticsSectionName: VET360.ANALYTICS_FIELD_MAP[fieldName],
+    blockEditMode: !!activeEditView,
+    activeEditView:
+      activeEditView === 'addressValidation'
```

Comment:
This ternary is to deal with an edge case: if the user is currently viewing the address validation view we need to handle things differently or text in the modal would be inaccurate. This is an unfortunate hack to get around an existing hack we've been using to determine if we need to show the address validation view or not.

Reply:
👍 We should also leave a comment in the code so this context does not yet lost